samples/tab-calendar/ts/README.md

```diff
@@ -0,0 +1,161 @@
+---
+page_type: sample
+description: This sample shows tab calendar in Outlook apps.
```

Comment:
This sample shows how to interact with Calendar in Outlook from Personal Tab

Reply:
Fixed